### ISSUE TITLE
Impement Rollback on Auto Eject

### ISSUE THREAD

**Issue by looperXD** [TotalFreedom/TotalFreedomMod#689]
When a Play get Auto Ejected for Nucker, or Breaking Blocks too fast, There is not Rollback to fix it, Admins have to manually do /rollback (username). If the nucker was used to Grief,  A rollback could undo it.

**Comment by Obi2** [comment]
I agree, but this should probably only be done on the final strike - sometimes there are false positives etc.

**Comment by ChargedCreeper** [comment]
@Obi2 That, and sometimes an op might accidentally active nuker due to the stupid keybinds of some clients.
